### ISSUE TITLE
could not determine kind of name for C.ydb_call_variadic_plist_func_st

### ISSUE THREAD

**Issue by TPTan96** [YottaDB/YDB#356]
My golang env
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/ubuntu/.cache/go-build"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/ubuntu/go"
GOPROXY=""
GORACE=""
GOROOT="/snap/go/3947"
GOTMPDIR=""
GOTOOLDIR="/snap/go/3947/pkg/tool/linux_amd64"
GCCGO="gccgo"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD=""
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build753087890=/tmp/go-build -gno-record-gcc-switches"

I have a problem when I go get lang.yottadb.com/go/yottadb

There is an error
could not determine kind of name for C.ydb_call_variadic_plist_func_st

**Comment by estess** [comment]
You did not mention what YDB version you were using or which branch of the Go wrapper but I'm going to guess you are trying to use the master branch of the Go wrapper in YDBGo and are the new r1.26 version of YDB. This combination does not work.

We are currently putting the finishing touches on the production version of the Go wrapper. That version is currently in the 'develop' branch of YDBGo. This mostly works with r1.26 but is the development branch so is being updated daily. When we release the YDBGo wrapper for production (and at that time move it into the master branch) it will require a new r1.28 version of YDB as we found some issues on the C side that were contributing to the instability of the Go wrapper.

Bottom line, if you want the latest and greatest, do your own build of YDB from the master branch and pull down the develop branch of YDBGo. If you just want to use a combo that works, stay with r1.24 and the master branch of YDBGo until the production level release of the Go wrapper on r1.28.

Note our development is over on gitlab instead of github so while we have a presence here on github, the real fun is over on on our gitlab site.
